assets/js/base/utils/errors.ts

```diff
@@ -0,0 +1,41 @@
+export interface ErrorObject {
```

Comment:
Move the ErrorObject here, it makes sense to be here where as a place for reuse imo.